### PR TITLE
Fixed #287, added `lookup()` in `Model`.

### LINKED ISSUE #287: Why the update() function will update all records of table?
* GINO version: 0.7.5
* Python version: 3.6.6
* asyncpg version: 0.17.0
* aiocontextvars version: 0.1.2
* PostgreSQL version: 9.6.9

### Description

I have wrote a function to update some information in database,but it not only update the specified record,but also all records of table.

### What I Did

```
Here is my codes.



async def change_game_informations(game_id, creator_openid, **kwargs):
    game = await Game.query.where(
        Game.game_id == game_id
    ).gino.first()
    if game.creator_openid == creator_openid:
        await game.update(**kwargs).apply()
        print(game.game_city)
        return game
    else:
        return False


```

### PR DESCRIPTION
**Pull request by fantix**
Refs #287

### CODE REVIEW

File: tests/test_crud.py [hunk 1]
```diff
@@ -237,3 +237,41 @@ class NameCard(db2.Model):
...
+        channel_id = db.Column(db.String(1), default='A')
+
+    await Game.gino.create()
+    try:
+        game_1 = await Game.create(game_id='1', channel_id='X')
+        game_2 = await Game.create(game_id='2', channel_id='Y')
+        uq = game_1.update(game_id='3')
+        with pytest.raises(TypeError,
+                           match='Model Game has no table, primary key'):
+            await uq.apply()
```
Comment (wwwjfy) [inline tests/test_crud.py]: minor: `uq` is not used anywhere else, no need to declare as a variable?

File: tests/test_crud.py [hunk 2]
```diff
@@ -237,3 +237,41 @@ class NameCard(db2.Model):
...
+                           match='Instance-level CRUD operations not allowed'):
+            await game_2.delete()
+        with pytest.raises(LookupError,
+                           match='Instance-level CRUD operations not allowed'):
+            await game_2.query.gino.all()
+        with pytest.raises(LookupError,
+                           match='Instance-level CRUD operations not allowed'):
+            await game_2.select('game_id')
+
+        assert game_1.game_id == '3'
```
Comment (wwwjfy) [inline tests/test_crud.py]: This is not intuitive when `uq.apply()` fails, one comment wouldn't harm. :)

File: tests/test_json.py [hunk 3]
```diff
@@ -96,6 +96,9 @@ class News(db.Model):
         profile = db.Column(JSON(), nullable=False, server_default='{}')
         visits = db.IntegerProperty(default=0)
 
+        def lookup(self):
+            return True
```
Comment (wwwjfy) [inline tests/test_json.py]: This should be discouraged, right? If so, let's not set a bad example in the test.

File: tests/test_crud.py (hunk 1 above)
Comment (fantix) [reply tests/test_crud.py]: ah it was to test `update()` doesn't raise

**Comment by fantix** [comment]
Thanks for reviewing!

**Pull request merged**